css-canvas-medical/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="top-container">
+  <div class="header">
+    <h1>Canvas</h1>
+  </div>
+  <div class="right-container">
+    <p class="top-right"> Product Implementation About Us Blog </p>
+    <button class="top-button"> Contact Us</button>
+  </div>
+</div>
+<hr>
+<div class="first-section">
```

Comment:
a `row` class for each of these sections might help distinguish each section separation, or I would make the naming convention consistent:
first-section, second-section, third-section, etc